src/status_im2/subs/wallet_2/networks.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im2.subs.wallet-2.networks
+  (:require [clojure.string :as string]
+            [re-frame.core :as re-frame]
+            [quo2.foundations.resources :as resources]))
+
+(re-frame/reg-sub
+ :wallet-2/filtered-networks-by-mode
+ :<- [:wallet-2/networks]
+ (fn [[networks] [_ test?]]
+   (let [filter-fn (if test? :Test :Prod)]
+     (map filter-fn
+          networks))))
+
+(def default-network-details
+  {:chain-id     1
+   :source       (resources/networks :ethereum)
+   :short-name   "eth"
+   :network-name :ethereum})
+
+(re-frame/reg-sub
+ :wallet-2/network-details
+ :<- [:wallet-2/filtered-networks-by-mode false]
+ (fn [networks]
+   (map
+    (fn [{:keys [chainId shortName chainName]}]
+      (let [network-name (keyword (string/lower-case chainName))]
+        (if (= 1 chainId)
```

Comment:
Although the chain IDs don't change, we should avoid the magic values. Instead of 1, it'd be better to refer to it by a constant like `constants/network-mainnet` or a keyword `:mainnet` like other places do like https://github.com/status-im/status-mobile/blob/a847f508f9367c3553fb41a18e492f3f4b4cf9ae/src/status_im/wallet/prices.cljs#L58

But then you'd need a conversion layer, so the constant is simpler.